src/FsCheck/Gen.fs

```diff
@@ -141,20 +141,19 @@ module Gen =
     [<CompiledName("Resize")>]
     let resize newSize (Gen m) = Gen (fun _ r -> m newSize r)
 
-    ///Generates a value with maximum size n.
+    ///Generates a value of the give size with the given seed.
     //[category: Generating test values]
     [<CompiledName("Eval")>]
-    let eval n rnd (Gen m) = 
-        let size,rnd' = range (0,n) rnd
-        m size rnd'
+    let eval size seed (Gen m) =
```

Comment:
No, the signature is the same as before. `int -> StdGen -> Gen<'T> -> 'T`. Just the name of the argument changed.